tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```
@@ -190,7 +190,8 @@ spec:
         if [ "$mustPublishIndexImage" = "true" ]; then
           echo "Index image will be published."
         elif [ "$fbc_opt_in" = "false" ]; then
-          echo "Index image will not be published because fbc_opt_in is set to false in Pyxis."
+          echo "Index image will not be published because fbc_opt_in of at least one referenced bundle"\
```

I think you are missing a space here between the 2 lines?

```suggestion
          echo "Index image will not be published because fbc_opt_in of at least one referenced bundle "\
```

---

When I ran the echo locally, if I put a space there it rendered an extra space in the output:

```
❯ echo "Index image will not be published because fbc_opt_in of at least one referenced bundle "\
     "image in the fragment is set to false in Pyxis."
Index image will not be published because fbc_opt_in of at least one referenced bundle  image in the fragment is set to false in Pyxis.
```

vs. when I leave out the space, it formats correctly:

```
❯ echo "Index image will not be published because fbc_opt_in of at least one referenced bundle"\
     "image in the fragment is set to false in Pyxis."
Index image will not be published because fbc_opt_in of at least one referenced bundle image in the fragment is set to false in Pyxis.
```

If that behavior is only local to my system then I'll change it, but otherwise I think the newline adds a whitespace char.

---

> If that behavior is only local to my system then I'll change it, but otherwise I think the newline adds a whitespace char.

No, it's ok. It's not about the newline. It's because for each argument to `echo` a space is added.

It would only be an issue, if you had all of the text as one long string argument:
```
echo "one two\
three"
one twothree
```

Even this is still an issue:
```
echo "one two"\
"three"
one twothree
```

Because effectively the `\` and the newline get removed so it's equivalent to `echo "one two""three"` which still has just one string argument.

But in your case because the second line in indented, those extra spaces separate the arguments and so a space is added.

I would recommend also adding a space before `\` to make it more explicit/clear, but that's a detail.